Build failure: open-music-kontrollers.sherlock

### Steps To Reproduce

Steps to reproduce the behavior:
1. install open-music-kontrollers.sherlock

### Build log

```
error: builder for '/nix/store/md6kjwlzv7v7n6jdb0l5plh8f1xghkn1-sherlock-0.28.0.drv' failed with exit code 1;
       last 10 log lines:
       > FAILED: sherlock_nk.so.p/sherlock_nk.c.o
       > gcc -Isherlock_nk.so.p -I. -I.. -I../props.lv2 -I../osc.lv2 -I../ser_atom.lv2 -Isubprojects/nk_pugl -I../subprojects/nk_pugl -I../subprojects/nk_pugl/pugl/include -I/nix/store/38dxy6xs6ms4rybn1c5z7pk2xvci9q4r-lv2-1.18.10-dev/include -I/nix/store/qj88pvxq3j6hvyy7clwmhzpffh67m9mm-sratom-0.6.14-dev/include/sratom-0 -I/nix/store/aplyv385h2d4yibzij4kg332iwvnmz1s-glu-9.0.2-dev/include -I/nix/store/syfid5mrc3w354m9dpsknhkz80c7a7pm-libglvnd-1.6.0-dev/include -I/nix/store/llyakg9ca1xc2y3gnlff7nfp6imxlday-glew-2.2.0-dev/include -I/nix/store/1xsnjfdqb7ha3jmy05nd437v3wmj887s-libX11-1.8.4-dev/include -I/nix/store/x5bim40wygc2847f3fz2c7g93p9c9fc2-xorgproto-2021.5/include -I/nix/store/b166iqd0yp2hakz4ablckf6izg7sdisc-libXext-1.3.4-dev/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=gnu11 '-DSHERLOCK_VERSION="0.28.0"' -D_GNU_SOURCE -fPIC -DPUGL_STATIC -fvisibility=hidden -MD -MQ sherlock_nk.so.p/sherlock_nk.c.o -MF sherlock_nk.so.p/sherlock_nk.c.o.d -o sherlock_nk.so.p/sherlock_nk.c.o -c ../sherlock_nk.c
       > In file included from ../sherlock_nk.h:25,
       >                  from ../sherlock_nk.c:33:
       > /nix/store/qj88pvxq3j6hvyy7clwmhzpffh67m9mm-sratom-0.6.14-dev/include/sratom-0/sratom/sratom.h:12:10: fatal error: serd/serd.h: No such file or directory
       >    12 | #include "serd/serd.h"
       >       |          ^~~~~~~~~~~~~
       > compilation terminated.
       > [10/15] Compiling C object sherlock.so.p/atom_inspector.c.o
       > ninja: build stopped: subcommand failed.
       For full logs, run 'nix log /nix/store/md6kjwlzv7v7n6jdb0l5plh8f1xghkn1-sherlock-0.28.0.drv'
```

### Notify maintainers

@magnetophon 

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.31, NixOS, 23.05 (Stoat), 23.05.20230602.f3d7434`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.13.3`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

Issue is also prevalent on `open-music-kontrollers.synthpod` package

```
error: builder for '/nix/store/85h46kafmsb5rwi8p4qgych4n3cx0rr7-synthpod-unstable-2021-10-22.drv' failed with exit code 1;
       last 10 log lines:
       >    12 | #include "serd/serd.h"
       >       |          ^~~~~~~~~~~~~
       > compilation terminated.
       > [71/158] Compiling C object sandbox_ui.lv2/libsbox_master.a.p/sandbox_master.c.o
       > [72/158] Compiling C object sandbox_ui.lv2/libsbox_slave.a.p/sandbox_slave.c.o
       > ../sandbox_ui.lv2/sandbox_slave.c: In function '_file_read':
       > ../sandbox_ui.lv2/sandbox_slave.c:323:9: warning: ignoring return value of 'read' declared with attribute 'warn_unused_result' [-Wunused-result]
       >   323 |         read(fd, buf, *len);
       >       |         ^~~~~~~~~~~~~~~~~~~
       > ninja: build stopped: subcommand failed.
       For full logs, run 'nix log /nix/store/85h46kafmsb5rwi8p4qgych4n3cx0rr7-synthpod-unstable-2021-10-22.drv'.
```

--- a/pkgs/development/libraries/audio/sratom/default.nix
+++ b/pkgs/development/libraries/audio/sratom/default.nix
@@ -43,6 +43,13 @@ stdenv.mkDerivation rec {
     "-Ddocs=disabled"
   ];
 
+  postFixup = ''
+    # remove once updated to 0.6.15 or above
+    for f in $dev/lib/pkgconfig/*; do
+      echo "Requires: lv2 >=  1.18.4, serd-0 >=  0.30.10, sord-0 >=  0.16.10" >> "$f"
+    done
+  '';
+
   passthru = {
     updateScript = writeScript "update-sratom" ''
       #!/usr/bin/env nix-shell

